components/drata/actions/create-asset/create-asset.mjs

```diff
@@ -0,0 +1,105 @@
+import drata from "../../drata.app.mjs";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/AssetsPublicController_createAsset/";
+
+export default {
+  key: "drata-create-asset",
+  name: "Create Asset",
+  description: `Create an asset. [See docs here.](${docsLink})`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    drata,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the asset",
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "The description of the asset",
+    },
+    assetClassTypes: {
+      type: "string[]",
+      label: "Asset Class Types",
+      description: "The asset class types",
+      options: [
+        "HARDWARE",
+        "POLICY",
+        "DOCUMENT",
+        "PERSONNEL",
+        "SOFTWARE",
+        "CODE",
+        "CONTAINER",
+        "COMPUTE",
+        "NETWORKING",
+        "DATABASE",
+        "STORAGE",
+      ],
+    },
+    assetType: {
+      type: "string",
+      label: "Asset Type",
+      description: "The asset type",
+      options: [
+        "PHYSICAL",
+        "VIRTUAL",
+      ],
+    },
+    ownerId: {
+      type: "integer",
+      label: "Owner ID",
+      description: "The ID of the owner",
+    },
+    notes: {
+      type: "string",
+      label: "Notes",
+      description: "Notes about the asset",
+      optional: true,
+    },
+    uniqueId: {
+      type: "string",
+      label: "Unique ID",
+      description: "Unique Id associated with this asset",
+      optional: true,
+    },
+    removedAt: {
```

Comment:
Can you note the required date format on all the date props? We really need an ISO string native type with a date picker for static values!

<img width="391" alt="Screenshot 2023-04-18 at 10 46 09 AM" src="https://user-images.githubusercontent.com/242668/232860850-2ab17128-2a80-4c52-837c-f98e5f8dded5.png">